schema/DoctrineMigrations/Version20231214112518.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231214112518 extends LoggableMigration
+{
+    public function getDescription(): string
+    {
+        return 'Added color into WebPortal';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE WebPortals ADD color VARCHAR(10) DEFAULT \'#000000\' NOT NULL ');
+
+        $colorMaps = [
+            'god' => '#2D333B',
+            'brand' => '#248475',
+            'admin' => '#0277BD',
+            'user' => '#BF360C',
+        ];
+
+        foreach ($colorMaps as $urlType => $color ) {
+            $this->addSql("UPDATE `WebPortals` SET color='$color' WHERE urlType='$urlType' AND userTheme <> 'default'");
```

Comment:
```suggestion
            $this->addSql("UPDATE `WebPortals` SET color='$color' WHERE urlType='$urlType'");
```
userTheme applies to old user portal and klearTheme to the rest